### PR TITLE
067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HM…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #932

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
+	  :Do(function(_,data) 
+	  	corr_id_second_req = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
+	  	
+	  	local function send_HMI_response()
+	  	  -- ATF issue: "Genivi: SDL doesn't accept some error_codes from HMI when they are sending with type of protocol_message "error""
+	  		self.hmiConnection:SendError(data.id, data.method, hmi_result_code[i].result_code) 
+	  	end
+	  	
+	  	RUN_AFTER(send_HMI_response, 1500)
```
Comment (OVikhrov) [inline test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua]: @istoimenova As I understand correct here you are responding for first (which should be successfuly) request, but after 1000ms you will get GENERIC_ERROR for first request

Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua]: @OVikhrov, I believe, here we are checking unsuccessful case
Successful one is verified in 066 script

Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua]: @OVikhor, To have GENERIC_ERROR SDL should wait default timeout from ini(10000 msec).
@dboltovskyi, yes we are checking unsuccessful case.

**Comment by istoimenova** [comment]
@OVikhrov, please check updates.

**Pull request merged**